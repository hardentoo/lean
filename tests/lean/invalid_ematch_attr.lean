#exit
constant f : ℕ → ℕ
axiom foo {m n : ℕ} : f m = m

attribute [ematch] foo

example : true := begin[smt] eblast end
