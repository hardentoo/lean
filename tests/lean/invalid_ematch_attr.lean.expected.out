invalid_ematch_attr.lean:1:0: warning: using 'exit' to interrupt Lean
